From 1bd9030eed93995a5641d14f3e9c577651588eb8 Mon Sep 17 00:00:00 2001
From: Adam Goldman <adamg@pobox.com>
Date: Sun, 26 Nov 2023 02:45:25 -0800
Subject: [PATCH] hwdb: ieee1394-unit-function: add Sony DVMC-DA1

This commit adds a hwdb entry for the Sony DVMC-DA1. This media converter
works with video capture software such as dvgrab, but it doesn't support
the AV/C command set and doesn't match the general entry.

(cherry picked from commit 6a42bdb37e3963f2d1e29814b714f1673fbbc595)

Related: RHEL-5950
---
 hwdb.d/80-ieee1394-unit-function.hwdb | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/hwdb.d/80-ieee1394-unit-function.hwdb b/hwdb.d/80-ieee1394-unit-function.hwdb
index e49e78f218..91e8149804 100644
--- a/hwdb.d/80-ieee1394-unit-function.hwdb
+++ b/hwdb.d/80-ieee1394-unit-function.hwdb
@@ -1348,6 +1348,16 @@ ieee1394:ven00080046mo00000000sp0000A02Dver00010001
  ID_MODEL_FROM_DATABASE=DCR-TRV310
  IEEE1394_UNIT_FUNCTION_VIDEO=1
 
+#
+# Media converter without AV/C command set.
+#
+
+ieee1394:node:ven0x080046mo0xfa0000units0x00a02d:0x010000
+ieee1394:ven00080046mo00FA0000sp0000A02Dver00010000
+ ID_VENDOR_FROM_DATABASE=Sony Corporation
+ ID_MODEL_FROM_DATABASE=DVMC-DA1
+ IEEE1394_UNIT_FUNCTION_VIDEO=1
+
 #
 # Node with multiple units for several functions.
 #
